pipelines/shortterm/transform_short.py

```diff
@@ -0,0 +1,135 @@
+"""Script to clean the raw plant data."""
+import pandas as pd
+from extract_short import fetch_all_plants
+
+
+def load_csv(filename: str) -> pd.DataFrame:
+    """Loads raw plant data csv into a pandas dataframe."""
+
+    return pd.read_csv(filename)
+
+
+def extract_nested_columns(raw_plants: pd.DataFrame) -> pd.DataFrame:
+    """Extracts the necessary data from the columns containing dictionaries."""
+
+    raw_plants_df = raw_plants.copy()
+
+    botanist = raw_plants_df['botanist'].apply(pd.Series)
```

Comment:
Can you not just use `.tolist`? Does this not do the same thing?

Might be wrong on this.

```suggestion
    botanist = pd.DataFrame(raw_plants_df['botanist'].tolist())
```

Reply:
I think technically it would but think it is safer to do it like my approach